.circleci/create_circleci_config.py

```diff
@@ -405,6 +406,7 @@ def job_name(self):
         "pip install -U --upgrade-strategy eager -r examples/pytorch/_tests_requirements.txt",
         "pip install -U --upgrade-strategy eager -e git+https://github.com/huggingface/accelerate@main#egg=accelerate",
     ],
+    pytest_num_workers=1,
```

Comment:
When `OMP_NUM_THREADS>1`, we should set this to `1`.

(well, since we have `dist=loadfile` in `pytest` option and we have only 2 files concerned, we won't see the problems. But let's not risk our future life ...)